api_generator/Cargo.toml

```diff
@@ -12,24 +12,24 @@ license = "Apache-2.0"
 anyhow = "1.0"
 array_tool = "1.0.3"
 dialoguer = "0.10.2"
-flate2 = "~1"
-globset = "~0.4"
+flate2 = "1"
+globset = "0.4"
 Inflector = "0.11.4"
 itertools = "0.11.0"
 lazy_static = "1.4.0"
 log = "0.4.8"
 path-slash = "0.2.1"
-proc-macro2 = "~1.0"
-quote = "~1.0"
+proc-macro2 = "1.0"
+quote = "1.0"
 reduce = "0.1.2"
 regex = "1.3.1"
-reqwest = { version = "~0.11", features = ["blocking"] }
+reqwest = { version = "0.11", features = ["blocking"] }
 semver = "1.0.14"
-serde = { version = "~1", features = ["derive"] }
+serde = { version = "1", features = ["derive"] }
 serde_json = "~1"
 simple_logger = "4.0.0"
-syn = { version = "~2.0", features = ["extra-traits", "full"] }
-tar = "~0.4"
+syn = { version = "2.0", features = ["extra-traits", "full"] }
+tar = "0.4"
```

Comment:
* syn widens from `2.0.*` to `2.*.*`, should be okay due to semver and syn hasn't hit `2.1` yet
* tar remains equivalent